src/modules/book/book.routes.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const { body } = require('express-validator');
+
+const isAuth = require('../../middleware/auth.middleware');
+
+const router = express.Router();
+
+const bookController = require('./book.controller');
+
+router.post('/create', [
+    body('name')
```

Comment:
Create a separate middleware for this under 'book/middleware' and develop a middleware specifically for preprocessing operations.